luwak/src/main/java/uk/co/flax/luwak/termextractor/QueryAnalyzer.java

```diff
@@ -2,14 +2,16 @@
 
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
+import java.util.function.ToDoubleFunction;
```

Comment:
@romseygeek there are these unused imports for ToDoubleFunction in a few classes